### PR TITLE
re-export PolarLineRenderer and PolarMapper from chaco.api

### LINKED ISSUE #629: Re-export PolarLineRenderer and PolarMapper from chaco.api
Ref [`PolarLineRenderer`](https://github.com/enthought/chaco/blob/c758a65222a2caef9a3d2c074ae9dd5775a3a77e/chaco/polar_line_renderer.py), [`PolarMapper`](https://github.com/enthought/chaco/blob/c758a65222a2caef9a3d2c074ae9dd5775a3a77e/chaco/polar_mapper.py)

### PR DESCRIPTION
**Pull request by aaronayres35**
closes #629 

This PR simply re-exports the 2 classes from the api module.

It also pulls a comment about `PolarMapper` currently just being LinearMapper into the docstring so it appears on api docs.  Eventually we will want to update it, see #47.  In the meanwhile it is fine to like in the api as long as this is documented.
(Actually looking at it, it looks like some changes have been made to LinearMapper that weren't copied over... I may go ahead and pull those directly so that the PolarMapper really does just match the LinearMapper like it says).
EDIT: done. The PolarMapper is now the current LinearMapper rather than an old version of the LinearMapper.

### CODE REVIEW

File: chaco/polar_mapper.py [hunk 1]
```diff
@@ -29,8 +29,11 @@ class PolarMapper(AbstractMapper):
     # Private traits
     # ------------------------------------------------------------------------
 
-    _scale = Float(1.0)  # number of screen space units per data space unit
+    # Number of screen space units per data space unit.
```
Comment (rahulporuri) [inline chaco/polar_mapper.py:32]:
```suggestion
    #: Number of screen space units per data space unit.
```

File: chaco/polar_mapper.py [hunk 2]
```diff
@@ -29,8 +29,11 @@ class PolarMapper(AbstractMapper):
...
+    _scale = Float(1.0)
+    # Is the range of the screen space empty?
```
Comment (rahulporuri) [inline chaco/polar_mapper.py:34]:
```suggestion
    #: Is the range of the screen space empty?
```

File: chaco/polar_mapper.py [hunk 3]
```diff
@@ -29,8 +29,11 @@ class PolarMapper(AbstractMapper):
...
     _null_screen_range = Bool(False)
+    # Is the range of the data space empty?
```
Comment (rahulporuri) [inline chaco/polar_mapper.py:36]:
```suggestion
    #: Is the range of the data space empty?
```

File: chaco/polar_mapper.py [hunk 4]
```diff
@@ -40,27 +43,38 @@ class PolarMapper(AbstractMapper):
...
-        values from data space into screen space.
+        Overrides AbstractMapper. Maps values from data space into screen space.
         """
         self._compute_scale()
         if self._null_data_range:
-            return array([self.low_pos] * len(data_array))
+            if isinstance(data_array, (tuple, list, ndarray)):
+                return full_like(data_array, self.low_pos, dtype=float64)
+            else:
+                return array([self.low_pos])
```
Comment (rahulporuri) [inline chaco/polar_mapper.py:53]: I'm not sure why these changes were made. Were these changes copied over from the linear mapper?

Comment (rahulporuri) [reply chaco/polar_mapper.py:53]: I have the same comment about the changes in `map_data` and `_compute_scale`

File: chaco/polar_mapper.py (hunk 1 above)
Comment (aaronayres35) [reply chaco/polar_mapper.py:32]: This are privately named traits, but I agree they do seem useful to have in the api docs. I will go ahead with the # to #: changes

Comment (aaronayres35) [reply chaco/polar_mapper.py:32]: Actually, I'm having second thoughts.  I don't think users should need to ever touch these traits correct?

**Review by rahulporuri — APPROVED** [review]
LGTM. I haven't tested to check that the example still works.
